### PR TITLE
hack/update: kicbase version

### PR DESCRIPTION
**Pull request by prezha**
fixes: #9464
fixes: #9420
replaces: #9439

pr #9464 ("automate update processes") split: part 4/4 includes: `hack/update/kicbase_version/update_kicbase_version.go`

### CODE REVIEW

**Comment by sharifelgamal** [comment]
I think in breaking up your update PRs, some sort of dependency broke, is there a PR blocking this one?

**Comment by prezha** [comment]
the conflicts are due to the pr #9464 (the original one - before i've split it into several prs) got merged first, and i had a couple of changes made in those individual prs; essentially my fault, i should have had closed the original pr right after the split (but left it just in case anyone wanted to compare)...
i've fixed this one now, should be ok to merge

**Review by tstromberg — APPROVED** [review]

**Comment by tstromberg** [comment]
Thank you!

**Pull request merged**
